src/main/java/com/vention/stock_market_share/repository/UserRepository.java

```diff
@@ -133,6 +134,10 @@ private void setPreparedStatementParameters(PreparedStatement preparedStatement,
         preparedStatement.setString(3, user.getEmail());
         preparedStatement.setInt(4, user.getAge());
     }
+
```

Comment:
And also delete setPreparedStatementParameters method. It’s not clear why you need to create a method to simply transfer parameters

Reply:
Deleted